app/src/lang/translations/de-DE.yaml

```diff
@@ -181,6 +181,7 @@ permissions_for_role: 'Elemente, die {role} Rolle {action} kann.'
 fields_for_role: 'Felder die von der {role} Rolle {action} werden können.'
 validation_for_role: 'Feld {action} Regeln, welche die {role} Rolle befolgen muss.'
 presets_for_role: 'Standardwert für die {role} Rolle.'
+presets_field_warning: 'Der App-Editor für Feld {field} unterstützt nur die detaillierte Syntax.'
```

Comment:
```suggestion
```

From a guess from the `App-Editor` here (as I'm not familiar with the language), I believe it's now outdated due to https://github.com/directus/directus/pull/16560/commits/3911a17610dbdc9bd5e804e1d67d4cf5763ceace 

That being said, opting to remove this in favor of follow up translations by users via Crowdin as the preferred workflow 👍